17.md

```diff
@@ -123,6 +123,11 @@ This NIP offers the following privacy and security features:
 
 The main limitation of this approach is having to send a separate encrypted event to each receiver. Group chats with more than 100 participants should find a more suitable messaging scheme.
 
+## NIP-19 entity
+To create an `nchat` link:
+  - `kind`: `17`
+  - `special`: one or more pubkeys concatenated
```

Comment:
I think we should not concatenate but add all pubkey hexes in a TLV list (which is already been used for relays).

May need to note that the order of keys doesn't matter.

Reply:
We could have multiple `special` entries but idk if thats a problem for some clients

Reply:
It's going to be hard to trust people do this correctly, but we can request the list to be in ascending order by pubkeys.

But I would parse and resort it myself before doing anything with it.